requirements/dev.txt

```diff
@@ -4,6 +4,7 @@ autopep8==1.2.4
 coverage==4.2
 coveralls==1.1
 django-autofixture==0.12.1
+django-extensions==1.7.8
```

Comment:
yeah, actually the django custom management command calls `runscript` (which is available in `django-extensions` app) to run `scripts/seed.py`. IMO, this was the easiest way to do this than to struggle through adding arguments to django management command. Also, I think this `runscript` would be coming handy in the near future.